lara-typescript/src/section-authoring/components/section-item-picker.tsx

```diff
@@ -71,19 +86,18 @@ export const SectionItemPicker: React.FC<IProps> = (props) => {
   const setItemClasses = (isSelectedItem: boolean) => {
     const classes = classNames("assessmentItemOption", {
       selected: isSelectedItem,
-      disabled: !isSelectedItem && currentSelectedItem !== undefined
+      disabled: false
```

Comment:
If this is always going to be false, maybe we don't need that class name for item buttons?